ports/stm32/mpconfigport.h

```diff
@@ -52,6 +52,7 @@
 #define MICROPY_OPT_COMPUTED_GOTO   (1)
 #define MICROPY_OPT_CACHE_MAP_LOOKUP_IN_BYTECODE (0)
 #define MICROPY_OPT_MPZ_BITWISE     (1)
+#define MICROPY_OPT_FAST_FACTORIAL  (1)
```

Comment:
It would be clearer to call this `MICROPY_OPT_MATH_FACTORIAL`, since the word "OPT" already means that enabling this will optimise the feature.